src/devices/tpms_truck.c

```diff
@@ -78,15 +77,17 @@ static int tpms_truck_decode(r_device *decoder, bitbuffer_t *bitbuffer, unsigned
 
     /* clang-format off */
     data_t *data = data_make(
-            "model",            "",             DATA_STRING, "Truck",
-            "type",             "",             DATA_STRING, "TPMS",
-            "id",               "",             DATA_STRING, id_str,
-            "wheel",            "",             DATA_INT,    wheel,
-            "pressure_kPa",     "Pressure",     DATA_FORMAT, "%.0f kPa",    DATA_DOUBLE, (float)pressure,
-            "temperature_C",    "Temperature",  DATA_FORMAT, "%.0f C",      DATA_DOUBLE, (float)temperature,
-            "state",            "State?",       DATA_FORMAT, "%x",          DATA_INT,    state,
-            "flags",            "Flags?",       DATA_FORMAT, "%x",          DATA_INT,    flags,
-            "mic",              "Integrity",    DATA_STRING, "CHECKSUM",
+            "model",          "",               DATA_STRING, "Truck",
+            "type",           "",               DATA_STRING, "TPMS",
+            "id",             "",               DATA_STRING, id_str,
+            "wheel",          "",               DATA_INT,    wheel,
+            "pressure_kPa",   "Pressure",       DATA_FORMAT, "%.0f kPa",    DATA_DOUBLE, (float)pressure,
+            "temperature_C",  "Temperature",    DATA_FORMAT, "%.0f C",      DATA_DOUBLE, (float)temperature,
+            "pressure_alert", "Pressure Alert", DATA_COND, flags & 0x4, DATA_STRING, "pressure alert",
```

Comment:
Best to prepare this value as var and use `"pressure_alert", "Pressure Alert", DATA_COND, pressure_alert, DATA_INT, pressure_alert,`